projects/pastanaga-angular/src/lib/chips/chip/chip.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { Observable } from 'rxjs';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+
+@Component({
+    selector: 'pa-chip',
+    templateUrl: './chip.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ChipComponent {
+    @Input() contact?: { name: string; image?: Observable<Blob> };
+
+    @Input() autoBackgroundAvatar = false;
+
+    @Input() set noCloseButton(value: boolean) {
+        this.canClose = !coerceBooleanProperty(value);
+    }
+
+    canClose = true;
+
+    @Input() disabled = false;
```

Comment:
it's better to use getters and setters here so you can use `coerseBooleanProperty`